indexers/src/utils/price.helpers.ts

```diff
@@ -0,0 +1,17 @@
+import { UniswapV2PairAbi__factory } from "../types/contracts";
+import { UNISWAP_USDC_ETH_CONTRACT_ADDRESS } from "../constants";
+
+/**
+ * Get the current price of Ethereum in USD
+ */
+export const getCurrentEthPriceInUsd = async (): Promise<number> => {
+  const uniswapUsdcContract = UniswapV2PairAbi__factory.connect(
+    UNISWAP_USDC_ETH_CONTRACT_ADDRESS,
```

Comment:
We should get the UNISWAP address on all the chains we support
Arbitrum, Base, Optimism, ETH and Bsc, unless it's the same address on all the chains

Reply:
This has been addressed. We've switched to using ChainLink's price feed contracts for each chain